src/mongo/MongoScrapbook.ts

```diff
@@ -201,7 +201,7 @@ class FindMongoCommandsVisitor extends MongoVisitor<MongoCommand[]> {
 	}
 
 	visitFunctionCall(ctx: mongoParser.FunctionCallContext): MongoCommand[] {
-		if (ctx.parent instanceof mongoParser.CommandContext) {
+		if (ctx.parent instanceof mongoParser.CommandContext && ctx._FUNCTION_NAME) {
```

Comment:
Not sure we should be avoiding creating a command here.  Need to play around with it.